secure_message/application.py

```diff
@@ -93,24 +97,18 @@ def handle_exception(error):  # NOQA pylint:disable=unused-variable
 
     return app
 
+def refresh_client_token_if_required(app):
```

Comment:
The tests aren't passing because styles check picks up wrong number of blank lines here